internal/introspect/introspect.go

```diff
@@ -69,10 +69,11 @@ type timeClaims struct {
 
 // ValidNow returns true if the expired and not before claims indicate that the token is valid for the current local
 // time. From RFC7519:
```

Comment:
Was this intentional? Just trying to keep formatting changes to a minimum.

Reply:
This error only comes up when you run `golangci-lint run` using _.golangci.yml_:
```internal/introspect/introspect.go:72: File is not `gofmt`-ed with `-s` (gofmt)```

I understand GitHub only runs _.github/workflows/golangci-lint.yml_, but thought it would be best to fix. Let me know if we should revert the formatting